addons/mail/static/src/models/call_main_view.js

```diff
@@ -148,6 +154,13 @@ registerModel({
             related: 'callView.channel',
             required: true,
         }),
+        /**
+         * Determines if the controller is an overlay or a bottom bar.
+         */
+        isControllerFloating: attr({
+            default: false,
+            compute: '_computeIsControllerFloating',
+        }),
```

Comment:
**Not to change in this PR**
Just observation for future improvements:

`CallMainView/isControllerFloating` looks like a poor concept, which in turn makes overall code hard to understand.

I think everything should be just `CallMainView/showOverlay`.
Just on terminology, "controller floating" is just a similar and inferior way to say "overlay".
However, the compute is not the same as different, and this doesn't make sense for me.

Of course, some other changes should be made, like renaming `showOverlayTimer` to something like `visibleCallActionListTimer`, so that it's clear that it's a timer to determine how long overlay is visible and then become invisible. And this also removes the bad wording that this timer is coupled with "overlay" mode, which is the root cause of poorly defining "overlay" in the 1st place, and itself introducing this weird and inappropriate different concept "floating controller".